resources/js/Pages/Guest/ForgotPassword.vue

```diff
@@ -2,25 +2,33 @@
 import { useForm } from '@inertiajs/vue3'
 import CustomInput from '@/components/Common/CustomInput.vue'
 import { Head } from '@inertiajs/vue3'
+import { inject, onMounted, type Ref } from 'vue'
+import type Header from '@/components/Common/Header.vue'
 
-defineProps<{
-  errors: Record<string, string>
-}>()
-
+const headerRef = inject<Ref<InstanceType<typeof Header>>>('header')
+onMounted(() => {
+  if (headerRef)
+    headerRef.value.titleRef = 'Przypomnij hasło'
+})
 const form = useForm({ email: '' })
 
 function submit() {
-  form.post('/auth/forgot-password')}
+  form.post('/auth/forgot-password')
+}
 </script>
 
 <template>
   <Head>
-    <title>Przypomnij hasło</title>
+    <title>{{ headerRef?.titleRef }}</title>
```

Comment:
Why ? You can just `<Head title="Home">`. I think meta tags is not necessary here. But if you want to keep meta tags, change please title set to some more readable.

Reply:
Fixed